shell/testing/tester_main.cc

```diff
@@ -104,12 +109,31 @@ int RunTester(const flutter::Settings& settings, bool run_forever) {
 
   auto current_task_runner = fml::MessageLoop::GetCurrent().GetTaskRunner();
 
+  auto threadhost = std::make_unique<ThreadHost>(
```

Comment:
This will create (but not use) the extra threads even if the multi-threaded specification is absent. Please create the thread host only when needed.